br/pkg/lightning/errormanager/errormanager.go

```diff
@@ -148,6 +148,20 @@ const (
 		ORDER BY raw_key;
 	`
 
+	selectDataConflictKeysReplace = `
+		SELECT raw_key
+		FROM %s.` + ConflictErrorTableName + `
+		WHERE table_name = ? AND index_name = 'PRIMARY'
```

Comment:
and seems we should add more INDEX covering the WHERE clause to `ConflictErrorTableName`